docs/ru/5-templates-syntax.md

```diff
@@ -64,7 +64,7 @@ elem(name)
 ```js
 /**
  * @param {String} modName имя модификатора блока
- * @param {String|Boolean} modVal значение модификатора блока
+ * @param {String|Boolean|'*'} [modVal] значение модификатора блока
```

Comment:
`'*'` in `i-bem` also means changing modifier value to empty (string) or `false`. Here we don't need it and probably we shouldn't add `'*'` since we don't need it for now but we can't be sure about future.